app/build.gradle

```diff
@@ -8,6 +8,7 @@ apply plugin: 'checkstyle'
 android {
     compileSdkVersion 25
     buildToolsVersion "25.0.3"
+    testBuildType "beta"
```

Comment:
I saw that we can configure the build variant to run tests on in buddybuild. Do we even need this in the gradle file?